lib/common/xml.c

```diff
@@ -3787,7 +3787,8 @@ dump_xml_element(xmlNode * data, int options, char **buffer, int *offset, int *m
     if (data->children) {
         xmlNode *xChild = NULL;
 
-        for (xChild = __xml_first_child(data); xChild != NULL; xChild = __xml_next(xChild)) {
+        //for (xChild = __xml_first_child(data); xChild != NULL; xChild = __xml_next(xChild)) {
+        for (xChild = data->children; xChild != NULL; xChild = xChild->next) {
```

Comment:
Given that 59c2413147adab0a9d586bf7a29e2b5710d73552 has brought back the ability of processing XML comments, I guess some  people want to store comments in the cib ;-)